events/elections/2018/README.md

```diff
@@ -0,0 +1,65 @@
+# 2018 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
+
+## Purpose
+
+The role of this election is to fill out the three(3) seats due for reelection this year on the [Kubernetes Steering Committee](https://github.com/kubernetes/steering).
+
+## Voting
+
+This election will  shape the future of Kubernetes as a project and community. You will be voting on three (3) seats up for election.
+Every elected community member will serve a two year term.
+
+The candidates you are voting on will be meeting regularly to strategically grow the project with contributors in mind. Most of the technical decisions, architecture planning, and the like come out of SIGs and other working groups.
+Some examples of responsibilities to consider as you are voting:  
+-  Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Kubernetes.
+-  Charter and refine policy for defining new community groups (including Special Interest Groups, Working Groups, and Committees), and establish transparency and accountability policies for such groups.  
+-  Define, evolve, and defend a Code of Conduct, which must include a neutral, unbiased process for resolving conflict
+
+Full scope of responsibilities, goals, and/or future election timelines, see [steering committee charter](https://github.com/kubernetes/steering/blob/master/charter.md).
+
+For more context, please see the [current issues](https://github.com/kubernetes/steering/blob/master/backlog.md) that need to be resolved or a previous governance [meeting video](https://www.youtube.com/watch?v=ltRKXLl0RaE&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ&index=23) which led to this whole process.
+
+## Process
+
+Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](http://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). 
+The top vote getters, modulo the corporate diversity requirement of no more than 1/3 of the committee from a single company, will be elected to the respective positions.
+
+You will be ranking your choices 1-20 with an option for no opinion. In the event of a tie, a coin will be flipped.
+
+The election will open for voting on September 19, 2018 at 04:00pm UTC and end two weeks after on October 3, 2018 at 01:00am UTC. You will receive an email to the address on file at the start of the election from Jorge Castro (jorge@heptio), Community Manager, please whitelist if necessary. Detailed voting instructions will be addressed in email and the CIVS polling page.
+
+### Schedule
+
+August 15 - Announcement of Election
+September 14 - All candidate bios / statement of intents due
+September 19 - Election Begins
+October 3 - Election Closes
+October 4 - Announcement of Results
+
+### Eligibility  
+Members of Standing are defined by the union of:
+* SIG leads
```

Comment:
SIG Chairs/Leads (addition of the word chair)

Reply:
SIG/WG/Subproject leads/chairs.